internal/pkg/cli/job_deploy.go

```diff
@@ -0,0 +1,131 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"fmt"
+
+	"github.com/aws/copilot-cli/internal/pkg/aws/sessions"
+	"github.com/aws/copilot-cli/internal/pkg/config"
+	"github.com/aws/copilot-cli/internal/pkg/manifest"
+	"github.com/aws/copilot-cli/internal/pkg/term/command"
+	termprogress "github.com/aws/copilot-cli/internal/pkg/term/progress"
+	"github.com/aws/copilot-cli/internal/pkg/term/selector"
+	"github.com/aws/copilot-cli/internal/pkg/workspace"
+	"github.com/spf13/cobra"
+)
+
+type deployJobVars struct {
+	*GlobalOpts
+	Name         string
+	EnvName      string
+	ImageTag     string
+	ResourceTags map[string]string
+}
+
+type deployJobOpts struct {
+	deployJobVars
+
+	store        store
+	ws           wsSvcDirReader
+	unmarshal    func(in []byte) (interface{}, error)
+	cmd          runner
+	sessProvider sessionProvider
+
+	spinner progress
+	sel     wsSelector
+
+	// imageBuilderPusher imageBuilderPusher
+	// s3           artifactUploader
+	// addons       templater
+	// appCFN       appResourcesGetter
+	// jobCFN       cloudformation.CloudFormation
+
+	// // cached variables
+	// targetApp         *config.Application
+	// targetEnvironment *config.Environment
+	// targetWorkload    *config.Service
```

Comment:
Can we remove commented out code in the codebase? Even if we think we'll be adding them back pretty soon.

Personally, whenever I ran into commented out code that was left by another engineer I never felt comfortable removing it because I didn't know whether they intended to keep it there for some reason. Deleting commented code removes that confusion.